torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -178,23 +217,78 @@ def _fill_tensor_meta(
     for i, (onnxscript_value, expected_value) in enumerate(
         zip(flat_onnxscript_values, flat_expected_values)
     ):
-        # Only set shape for now as we don't need type information.
-        onnxscript_value.shape = tuple(expected_value.size())
-        # FIXME(titaiwang): type promotion is not supported yet,
-        # so set dtype will break onnx shape inference.
+        # We set node output sizes to be dynamic to continue the model conversion,
+        # and inputs are also set to be dynamic in add_input().
+        onnxscript_value.shape = tuple(
+            [dim if isinstance(dim, int) else None for dim in expected_value.size()]
+        )
+        # TODO(titaiwang): This could break in terms of lacking of type_promotion now.
         onnxscript_value.dtype = expected_value.dtype
         if i > 0:
             onnxscript_value.name = f"{name}_{i}"
         else:
             onnxscript_value.name = name
 
 
+def _wrap_fx_args_as_torch_args(
+    complete_args: list, complete_kwargs: dict
+) -> Tuple[tuple, dict]:
```

Comment:
nit: similar to above comment on type annotations. Would greatly help if annotated with more details. e.g. `Tuple[Tuple[torch.Tensor, ...], Dict[str, torch.Tensor]]`

Reply:
I introduced Annotation from torch/fx/node.py, and put it into _type_utils.py. LMK if it's good enough.